apps/argparse/argparse.hpp

```diff
@@ -741,6 +742,20 @@ class Argument {
     return *this;
   }
 
+  auto &store_into(std::set<std::string> &var) {
```

Comment:
yes, please try to upstream that. See https://github.com/p-ranav/argparse/pull/343 for a potential similar contribution. The tests have to be manually run with "test/tests" from the CMake build directory

Reply:
Done: https://github.com/p-ranav/argparse/pull/348